pkg/utils/pod/pod_utils.go

```diff
@@ -119,11 +125,24 @@ func AddLWSVariables(pod *corev1.Pod) error {
 		Value: fmt.Sprintf("%s-%s.%s.%s", lwsName, groupIndex, lwsName, pod.ObjectMeta.Namespace),
 	}
 
+	size, fount := pod.Annotations[leaderworkerset.SizeAnnotationKey]
+	if !fount {
+		return fmt.Errorf("Failure constructing environment variables, no size annotation found for pod %v", pod.Name)
```

Comment:
```suggestion
		return fmt.Errorf("Failure constructing environment variables, no size annotation found for pod %v", pod)
```

Technically the pods can have the same name if they are in a different namespace.

Reply:
thanks. i use `klog.KObj(pod)`.